signadot/operator/README.md

```diff
@@ -66,14 +67,19 @@ kubectl delete ns signadot
 | `serviceLabels`      | Labels to add to all deployed `Service` objects           | `{}`     |
 | `serviceAnnotations` | Annotations to add to all deployed `Service` objects      | `{}`     |
 
+
 ### Controller Manager parameters
 
 | Name                            | Description                                                                                               | Default |
 | ------------------------------- | --------------------------------------------------------------------------------------------------------- | ------- |
 | `allowedNamespaces`             | Restrict the namespaces in which `signadot-controller-manager` will operate                               | `[]`    |
+| `operator.replicas`             | Number of replicas for `signadot-controller-manager` deployment                                           | `2`     |
 | `sandboxTrafficManager.enabled` | Whether to enable the Sandbox Traffic Manager Sidecar on forked workloads                                 | `true`  |
 | `allowOrphanedResources`        | Allow Signadot Custom Resources to exist in the cluster when not created or managed via the control plane | `false` |
 
+ℹ️ For development clusters (such as Minikube, MicroK8s, or K3s), we recommend
```

Comment:
I would add a note here that much of the functionality is not in fact replicated in parallel when replicas > 1.

Reply:
Fixed